src/app/pages/common/entity/entity-table/entity-table-add-actions.component.ts

```diff
@@ -5,15 +5,13 @@ import { fromEvent as observableFromEvent } from 'rxjs';
 import { debounceTime, distinctUntilChanged } from 'rxjs/operators';
 import { TranslateService } from '@ngx-translate/core';
 
-import { EntityTableComponent } from './entity-table.component';
-
 @Component({
   selector: 'app-entity-table-add-actions',
   templateUrl: './entity-table-add-actions.component.html',
 })
 export class EntityTableAddActionsComponent implements OnInit, AfterViewInit, OnDestroy, OnChanges {
   @ViewChild('filter', { static: false }) filter: ElementRef;
-  @Input('entity') entity: EntityTableComponent;
+  @Input('entity') entity: any; // EntityTableComponent;
```

Comment:
Well, I don't know if this is better. I'll take circular dependency for now over any :)